### PR TITLE
Migrated test_mount_after_removing_client_logs_dir

### PR DESCRIPTION
**Pull request by nik-redhat**
Migrated TC [test_mount_after_removing_client_logs_dir](https://github.com/gluster/glusto-tests/blob/master/tests/functional/glusterd/test_mount_after_removing_client_logs_dir.py)

Fixes: #558 
Updates: #292 
Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
Test run:
```
Summary:
+---------------+-------+--------------+
|    Category   | Cases | Pass Percent |
+---------------+-------+--------------+
| nonDisruptive |   0   |      0       |
|   Disruptive  |   1   |    100.0     |
|     Total     |   1   |    100.0     |
+---------------+-------+--------------+
Table:
 test_mount_after_removing_client_logs_dir
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|     dist    |     PASS    | 27.425090074539185 |
|   dist-rep  |     PASS    | 33.28997302055359  |
|     rep     |     PASS    | 25.348841190338135 |
|     disp    |     PASS    |  24.9003689289093  |
|  dist-disp  |     PASS    | 30.620460271835327 |
+-------------+-------------+--------------------+


Framework runtime : 142.57295441627502
```

File: common/ops/support_ops/io_ops.py [hunk 1]
```diff
@@ -110,6 +110,36 @@ def path_exists(self, list_of_nodes: list, list_of_paths: list) -> bool:
 
         return True
 
+    def get_dir_contents(self, path: str, node: str,
+                         recursive: bool = False,) -> list:
```
Comment (aujjwal-redhat) [inline common/ops/support_ops/io_ops.py:114]: nit: Do we need the extra , : )

File: tests/functional/glusterd/test_mount_after_removing_client_logs_dir.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along`
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+  Description:
+    A testcase to remove /var/log/glusterfs/ on client, mounting a volume
+    and createing a file and a dir on it.
```
Comment (aujjwal-redhat) [inline tests/functional/glusterd/test_mount_after_removing_client_logs_dir.py:20]: Indentation

**Review by aujjwal-redhat — COMMENTED** [review]
Btw, why does the test case in glusto have almost similar two functions ? : |

**Comment by nik-redhat** [comment]
> Btw, why does the test case in glusto have almost similar two functions ? : |

Those are 2 separate TCs :p

File: common/ops/support_ops/io_ops.py (hunk 1 above)
Comment (nik-redhat) [reply common/ops/support_ops/io_ops.py:114]: ack

File: tests/functional/glusterd/test_mount_after_removing_client_logs_dir.py (hunk 2 above)
Comment (nik-redhat) [reply tests/functional/glusterd/test_mount_after_removing_client_logs_dir.py:20]: ugghh

**Comment by aujjwal-redhat** [comment]
> > Btw, why does the test case in glusto have almost similar two functions ? : |
> 
> Those are 2 separate TCs :p

Oh. Ok. So will we also create 2 for the same?

**Comment by nik-redhat** [comment]
> > > Btw, why does the test case in glusto have almost similar two functions ? : |
> > 
> > 
> > Those are 2 separate TCs :p
> 
> Oh. Ok. So will we also create 2 for the same?

Didn't get you :thinking:

File: tests/functional/glusterd/test_mount_after_removing_client_logs_dir.py (hunk 2 above)
Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_mount_after_removing_client_logs_dir.py:20]: :sweat_smile:

**Comment by aujjwal-redhat** [comment]
> > > > Btw, why does the test case in glusto have almost similar two functions ? : |
> > > 
> > > 
> > > Those are 2 separate TCs :p
> > 
> > 
> > Oh. Ok. So will we also create 2 for the same?
> 
> Didn't get you

Aah... no silly confusion :sweat_smile: Kuch aur samajh liye

**Review by aujjwal-redhat — APPROVED** [review]
LGTM

**Review by nishith-vihar — APPROVED** [review]
LGTM

**Pull request merged**